dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/DefaultFuture.java

```diff
@@ -251,11 +251,11 @@ private String getTimeoutMessage(boolean scan) {
                 + (sent > 0 ? " client elapsed: " + (sent - start)
                 + " ms, server elapsed: " + (nowTimestamp - sent)
                 : " elapsed: " + (nowTimestamp - start)) + " ms, timeout: "
-                + timeout + " ms, request: " + (logger.isDebugEnabled() ? request : getRequestWithoutData()) + ", channel: " + channel.getLocalAddress()
+                + timeout + " ms, request: " + (logger.isDebugEnabled() ? request : getRequestWithoutData(request)) + ", channel: " + channel.getLocalAddress()
                 + " -> " + channel.getRemoteAddress();
     }
 
-    private Request getRequestWithoutData() {
+    private static Request getRequestWithoutData(Request request) {
```

Comment:
Don't use static.

Static results in many problems during the multi-instance refactor.

Reply:
would you mind explaining it? I think this useage is not perfect, but I don't understand the meaning of `Static results in many problems during the multi-instance refactor.`
please help me, thanks.

Reply:
I think private static method it's ok.